manual_flagging/flagging.py

```diff
@@ -423,13 +427,18 @@ def create_bins_and_labels(input_list):
 # Get sale.metadata table
 commit_sha = sp.getoutput("git rev-parse HEAD")
 
-# Write to sale.group_mean table
+run_type = (
+    "initial_flagging"
+    if not inputs["manual_update"]
+    else "manual_update_only_new_sales"
+    if inputs["manual_update_only_new_sales"]
+    else "manual_update"
+)
+
```

Comment:
I think this works well without any need for a migration or meta data restructure.